printer_mods/ArthurZheng/Foldable7InchDisplay/README.md

```diff
@@ -0,0 +1,64 @@
+# Foldable 7-Inch Display
+
+This design is modified from [revnull‘s rpi_7in_display_mount design](https://github.com/VoronDesign/VoronUsers/tree/732a9e28c78c44d65baffdcdb935d205e33d9897/printer_mods/revnull/rpi_7in_display_mount). Optimized the angle limit structure, and can use cheaper Chinese-made displays.
```

Comment:
can you change this link so it doesn't point to a specific commit in case revnull updates their mod in the future?

Reply:
Thanks for the reminder, I'll fix these and re pr.